src/QMCWaveFunctions/BsplineFactory/SplineC2C.h

```diff
@@ -63,6 +63,9 @@ class SplineC2C : public BsplineSet
   ///multi bspline set
   std::shared_ptr<MultiBspline<ST>> SplineInst;
 
+  ///Copy of original splines for orbital rotation
+  std::shared_ptr<std::vector<RealType>> coef_copy_;
```

Comment:
Maybe this is worth changing once SplineC2R is implemented, but at that point all of these SplineX2Y classes will be using these. Seems redundant to have it defined in each header, when it could just be protected in BsplineSet.h

Reply:
right now coef_copy_carries all the dimensions used by SplineC2R. I think it needs to be here for now. Once we clean up the encapsulation of the "old" coefficients, this should go away.